cmd/fanal/main.go

```diff
@@ -67,15 +68,15 @@ func run() (err error) {
 		SkipPing: true,
 	}
 
+	cleanup := func() {}
```

Comment:
What's this for?

Reply:
If we define `func` with nil, it will cause panic.
https://play.golang.org/p/bE51u0pk0Ko

But if we define `func` like the above, it works.
https://play.golang.org/p/uFaa4mhUAoF

Or, we have to do nil check.

```
defer func(){
    if cleanup != nil {
        cleanup()
    }
}
```

Reply:
I see. I understand.